scripts/kvtag_query_scale/template.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash -l
+
+#REGSBATCH -q regular
+#DBGSBATCH -q debug
+#SBATCH -N NODENUM
+#REGSBATCH -t 1:00:00
+#DBGSBATCH -t 0:30:00
+#SBATCH -C cpu
+#SBATCH -J JOBNAME
+#SBATCH -A m2621
```

Comment:
Please remove the project ID from this.

Reply:
Sure, I will leave it as a replacable variable in the template.